docs/en/reference/information_schema/load_tracking_logs.md

```diff
@@ -0,0 +1,16 @@
+---
+displayed_sidebar: "English"
+---
+
+# load_tracking_logs
+
+`load_tracking_logs` provides error logs of load jobs. This view is supported from StarRocks v3.0 onwards.
+
+The following fields are provided in `load_tracking_logs`:
+
+| **Field**     | **Description**                            |
+| ------------- | ------------------------------------------ |
+| JOB_ID        | The ID of the load job.                    |
+| LABEL         | The label of the load job.                 |
+| DATABASE_NAME | The database that the load job belongs to. |
+| TRACKING_LOG  | Error logs (if any) of the load job.       |
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/30f9da99-8102-42b5-84da-15ef4a7d980e)
-> The errors ...